com.github.rphys.yml

```diff
@@ -0,0 +1,101 @@
+app-id: com.github.rphys
```

Comment:
Cannot be `com.github`, you don't control that domain. See https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
"The Application ID should be a real URL of a domain that the app author has control over or where their app is hosted."

I don't control github, but it's where my app is hosted. I do not control any domains. Can I just not submit an app?

Reply:
> While a project may be hosted on GitHub or GitLab it does not have any control over the github.com or gitlab.com domain. Instead you should use io.github or io.gitlab as shown above.

Use `io.github.<app>`

Reply:
done